package.json

```diff
@@ -91,27 +90,33 @@
     "webpack-sources": "^1.1.0"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.19",
+    "@babel/preset-react": "^7.18",
+    "@babel/runtime": "^7.19",
+    "@sect/modernizr-loader": "^1.0.2",
```

Comment:
Some of these things probably should be in webpack config, but that can be sorted out later.